components/cam/bld/namelist_files/namelist_definition.xml

```diff
@@ -7333,6 +7333,12 @@ Radius of Earth [m].
 Default: set to shr_const value
 </entry>
 
+<entry id="omega" type="real" category="physconst"
```

Comment:
I think it would be worthwhile using a more descriptive variable name here, since "omega" could probably refer to a few dozen different quantities within the larger context of the model as a whole.